flowzone.yml

```
@@ -1617,12 +1620,20 @@ jobs:
           node-version: 18
 
       - name: Docusaurus Builder
-        uses: product-os/docusaurus-builder@master
+        if: |
+          needs.project_types.outputs.has_readme == 'true' &&
+          inputs.cloudflare_website != '' && inputs.custom_website_build != 'true'
```

```suggestion
          inputs.docusaurus_website == true
```
We should check `inputs.cloudflare_website` at the top of the job as per my above comment, and use a docusaurus boolean here.